devicemodel/samples/nuc/launch_vxworks.sh

```diff
@@ -16,20 +16,16 @@ fi
 mem_size=2048M
 
 # Note:
-#      Here we just launch VxWorks as a normal vm without lapic_pt. If you want try it
-#      with lapic_pt, you should add the following options and make sure the front-end virtio-console
-#      driver use polling mode (This feature is not supported by VxWorks offically now and should
-#      implement it by youself).
-#
-#      --virtio_poll 1000000 \
-#      --s 2, virtio-console,@pty:pty_port \
-#
-#      Once the front-end polling mode virtio-console get supported by VxWorks offically, we will
-#      add the lapic_pt option.
+#      For RTVM with lapic_pt, we only support virtio devices with polling mode enabled for both
+#      front-end and back-end. The virtio devices with polling mode are not supported by VxWorks
+#      offically now and should be implemented by yourself.
```

Comment:
do you have the FE driver in hand?

Reply:
Yes.

Reply:
then change the text to say:
The virtio devices with polling mode are not supported by VxWorks  officially now, and we are working on upstream the front-end drivers.

Reply:
Got it. Will update.

Reply:
> Yes.

That sentence confuses me a bit. It sounds like we have a driver available for this but it is not yet in the VxWorks package. If true, how can users get it? Are we distributing it?

Reply:
Yes, we hold the driver source code internally.
Actually, we are trying to figure out how to upstream our patches to VxWorks.

Reply:
In this case, we should also update the [tutorial](https://projectacrn.github.io/latest/tutorials/using_vxworks_as_uos.html) because that piece is missing from it. Are we able to share that code at all? If not, enabling this option by default is premature in my opinion.